simple_client_example.py

```diff
@@ -1,16 +1,13 @@
 """Simple test of new API Changes."""
 import asyncio
 import json
-import logging
 import pprint
 from datetime import date, timedelta
 
-import mytoyota.utils.logging.logging_config  # noqa # pylint: disable=unused-import
 from mytoyota.client import MyT
 from mytoyota.models.summary import SummaryType
 
 pp = pprint.PrettyPrinter(indent=4)
-_LOGGER = logging.getLogger(__name__)
```

Comment:
What if this is not imported through HA but used standalone? Don't we still need to invoke the logger somewhere? No nothing about it, so just a question

Reply:
The logger must be initiated by the programme that uses the lib. In the case of Home Assistant, it is the root logger of Home Assistant.
If you use the lib directly, e.g. via `simple_client_example.py`, the logger must be initiated directly in it if required. But we should leave that to the user.